### ISSUE TITLE
Request: Overscroll on all lists

### ISSUE THREAD

**Issue by avario** [ProxymanApp/Proxyman#543]
Can we please have overscroll (rubber-banding, bouncing) enabled on all lists in the app? This is mostly an issue on the sidebar and the main list of requests. Without overscroll it's not clear whether there are more entries in the list or if I have reached the end of the list. Sometimes it makes it feel like the app is just not responding to my scroll input.

On iOS this would be setting `UIScrollView.alwaysBounceVertical = true`. I'm not sure what the equivalent for macOS is

**Comment by NghiaTranUIT** [comment]
Hi @avario, let check out this build (https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_2.1.1_Allow_Bounce_Vertical.dmg), which enables the [verticalScrollElasticity](https://developer.apple.com/documentation/appkit/nsscrollview/1403475-verticalscrollelasticity)

![2020-07-08 14 06 18](https://user-images.githubusercontent.com/5878421/86888457-65418e80-c124-11ea-8e7a-5be53f175498.gif)

This effect only occurs if we scroll by the TouchPad. Scroll by mouse doesn't cause this effect. Similar to the Finder app.

**Comment by avario** [comment]
That looks great! Are you able to apply the same behaviour to the outline view on the left too? I think this would also be in line with how the outline view behaves in Finder.

**Comment by NghiaTranUIT** [comment]
Definitely. I will revisit all Lists and make sure they have this update 👍
